### PR TITLE
addListing page from clone project

### PR DESCRIPTION
**Pull request by trampham1104**

### CODE REVIEW

**Review by hthm00 — CHANGES_REQUESTED** [review]
The file is empty!

**Comment by trampham1104** [comment]
> The file is empty!

> Just update it, can you check again? Thank you 🤓

File: pages/createListing.html [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+
+      <!--Upload Image-->
+      <div class="input-group">
+        <span class="input-group-text" id="addon-wrapping">Upload Your Business Photo</span>
+        <input type="file" class="form-control">
+        <button type="button" class="btn btn-secondary">Upload Photo</button>
+      </div>
+
+      <!-- Save Draft & Publish Button-->
+      <button type="draft" class="btn btn-secondary">Save Draft</button>
```
Comment (hthm00) [inline pages/createListing.html:60]: There is no type "draft" in a button tag.

File: pages/createListing.html [hunk 2]
```diff
@@ -0,0 +1,65 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+  </head>
+
+  <body>
+  
+  <h1 class="center form-title">Add your Listing</h1>
+
```
Comment (hthm00) [inline pages/createListing.html:10]: Missing a form

File: pages/createListing.html [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+  <head>
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.3.0-alpha1/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-GLhlTQ8iRABdZLl6O3oVMWSktQOp6b7In1Zl3/Jr59b6EGGoI1aFkw7cmDA6j6gD" crossorigin="anonymous">
+  </head>
+
+  <body>
+  
+  <h1 class="center form-title">Add your Listing</h1>
+  
+   <div class="container">
+      <div class="enterinfo">
```
Comment (hthm00) [inline pages/createListing.html:12]: Class name is incorrect.

File: pages/createListing.html [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+          <span class="input-group-text" id="addon-wrapping">Enter your anana Name</span>
```
Comment (hthm00) [inline pages/createListing.html:13]: id field is not descriptive

**Review by hthm00 — CHANGES_REQUESTED** [review]
Overall, good work!

**Pull request closed without merging**